lib/reverse_words.rb

```diff
@@ -1,6 +1,21 @@
 # A method to reverse each word in a sentence, in place.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n + m/2) --> O(n) where n is the length of sentence and m is length of all words excluding spaces.
```

Comment:
Since the `do` loops are nested, I believe this is O(n<sup>2</sup>) since you are traversing the words and then traversing the characters.